crates/bevy_gltf/src/loader.rs

```diff
@@ -669,20 +675,23 @@ async fn load_image<'a, 'b>(
             if let Ok(data_uri) = DataUri::parse(uri) {
                 let bytes = data_uri.decode()?;
                 let image_type = ImageType::MimeType(data_uri.mime_type);
+                let mut image = Image::from_buffer(
+                    &bytes,
+                    mime_type.map(ImageType::MimeType).unwrap_or(image_type),
+                    supported_compressed_formats,
+                    is_srgb,
+                )?;
+                image.sampler_descriptor = sampler_descriptor;
```

Comment:
the sampler should probably be an argument to `Image::from_buffer` - it is manually set after calling in 3 of 4 callsites (and it's easy to construct the default variant in the remaining case).